Library/Formula/gneural-network.rb

```diff
@@ -0,0 +1,24 @@
+class GneuralNetwork < Formula
+  desc "GNU Gneural Network implements a programmable neural network."
+  homepage "https://www.gnu.org/software/gneuralnetwork/"
+  url "http://cvs.savannah.gnu.org/viewvc/*checkout*/gneuralnetwork/gneural_network-0.5.0.tar.gz?revision=1.1&root=gneuralnetwork"
```

Comment:
Seems to be legit checking out the upstream website but this is a very unusual URL for a GNU project. Hmm.